cmd/create/execution.go

```diff
@@ -132,6 +133,12 @@ The generated file would look similar to this. Here, empty values have been dump
   task: core.type_system.custom_objects.add
   version: v3
 
+9. When creating a new execution, you can assign it to a specific cluster pool:
+
+::
+
+   flytectl create execution --execFile execution_spec.yaml -p flytesnacks -d development --clusterPool my-gpu-cluster 
```

Comment:
Can we mention that this requires implementing the executor plugin and link to plugin interface